### ISSUE TITLE
what mean for result data add one and divide two 

### ISSUE THREAD

**Issue by LongMarch7** [switchablenorms/DeepFashion_Try_On#28]
hi，thank you for your work：
why the result data need add one and divide two

cv_img=(combine.permute(1,2,0).detach().cpu().numpy()+1)/2

**Comment by ChongjianGE** [comment]
Hi @LongMarch7 ,
This code is used for better visualization. You can visualize the output in any other way you like.
